ksql-engine/src/test/java/io/confluent/ksql/integration/SelectValueMapperIntegrationTest.java

```diff
@@ -38,17 +34,13 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import java.util.function.Function;
-import org.apache.kafka.connect.data.SchemaAndValue;
-import org.apache.kafka.connect.data.Struct;
 import org.junit.Rule;
 import org.junit.Test;
-import org.mockito.ArgumentCaptor;
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnit;
 import org.mockito.junit.MockitoRule;
 
-public class SelectValueMapperTest {
+public class SelectValueMapperIntegrationTest {
```

Comment:
Renaming this to `SelectValueMapperIntegrationTest`, since it's testing parsing, analysis, and execution of the mapper. I've taken the tests that make sense as unit tests and pulled them out into `SelectValueMapperTest`.